tools/selinux/icinga2.te

```diff
@@ -192,6 +193,9 @@ allow icinga2_t icinga2_port_t:tcp_socket name_bind;
 allow icinga2_t self:tcp_socket create_stream_socket_perms;
 corenet_tcp_connect_icinga2_port(icinga2_t)
 
+allow icinga2_t redis_port_t:tcp_socket name_connect;
```

Comment:
Use an interface instead of plain rules, so `redis_stream_connect` should be what you are looking for.

Reply:
Redis doesn’t provide such an interface. https://www.unix.com/man-page/centos/8/redis_selinux

Reply:
Ah, I see it is only available with EL8 and higher! Then it would be best to have to different policies, but I remember this was unwanted. Then still the code of the interface should be replicated: `stream_connect_pattern(icinga2_t, redis_var_run_t, redis_var_run_t, redis_t)`